src/camera.cc

```diff
@@ -2,6 +2,7 @@
 #include "ray.h"
 #include "scene.h"
 #include "scene_object.h"
+#include "point_light.h"
```

Comment:
btw point light shouldn't be needed here as scene contains vector of point lights. we should use those, this was just for testing